mesonbuild/envconfig.py

```diff
@@ -373,6 +374,10 @@ def is_irix(self) -> bool:
         """Machine is IRIX?"""
         return self.system.startswith('irix')
 
+    def is_baremetal(self) -> bool:
+        """Machine is bare metal?"""
+        return self.system.startswith('bare metal')
+
```

Comment:
I am struggling to understand the point of this addition.

Reply:
You are right, fixed now.